src/modules/app.js

```diff
@@ -0,0 +1,35 @@
+/* eslint no-undefined: "error" */
+
+import ListTemplate from './templateList.js';
+import localGet from './storage.js';
+
+const todoListContainer = document.querySelector('.toDoListItemContainer');
+
+const addList = (description, completed) => {
+  const listAdded = new ListTemplate(description, completed);
+  localGet.push(listAdded);
+  localStorage.setItem('listStorage', JSON.stringify(localGet));
+  window.location.reload();
+};
+
+window.displayList = () => {
+  todoListContainer.innerHTML = '';
+  localGet.forEach((item, id) => {
+    todoListContainer.innerHTML += `
+        <div class='toDoItem'>
+          <input class='item' id='check-${id}', completed)' type='checkbox' value=${item.completed}>
+          <input type='text' class='findInput' id='input-${id}' value=${item.description} />
+          <i onclick='updateList(${id})' class='fa-solid fa-file-pen' id='options-${id}'></i>
+          <i onclick='removeList(${id})'  class='fa-solid fa-trash' id='delete-${id}'></i>
+        </div>
+      `;
+  });
+};
+
+window.removeList = (index) => {
+  localGet.splice(index, 1);
+  localStorage.setItem('listStorage', JSON.stringify(localGet));
```

Comment:
Kindly note that after adding indexes to the list, when you remove an item from the list you need to update all the list indexes as well as described in the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#project-requirements) 🤗

![image](https://user-images.githubusercontent.com/73714615/193602893-67286a8a-f135-429b-ac40-5e243c95bc7f.png)